### PR TITLE
Some steps in tasks in the student guide have been updated.

### PR DESCRIPTION
**Pull request by ddobric**
Due to portal changes and some issues in the provisioned environment Some steps in tasks in the student guide have been updated.

### CODE REVIEW

**Comment by kylebunting** [comment]
@DawnmarieDesJardins, this pull request contains changes to make notes about differences when running the lab using a pre-provisioned environment. From previous conversations, I was under the impression that we were keeping the lab guides focused on provisioning using the associated ARM templates and setup steps.

Has the stance on this changed and do we want those kinds of changes coming into the labs?

If so, these changes look good. Otherwise, I think we should leave them out.

**Comment by DawnmarieDesJardins** [comment]
@kylebunting - Thanks for the review Kyle, you are correct.

@ddobric - Damir - Thank you for reaching out!  Due to the broad reach of our workshops, they are written as if there is no pre-provisioned environment.  If students are using a partner environment to work through the lab, they would be given additional instructions as needed by the lab hosts.

Closing PR with no further action.

**Comment by ddobric** [comment]
But changes in this PR also contain many other issues that are not related to the provisioned environment.

**Comment by timahenning** [comment]
@ddobric I have Kyle to review the changes.

File: Hands-on lab/HOL step-by-step - App modernization.md [hunk 1]
```diff
@@ -396,7 +396,9 @@ In this task, you use the Azure Cloud shell to retrieve the IP address of the Sq
 
    ![In the You have no storage mounted dialog, a subscription has been selected, and the Create Storage button is highlighted.](media/cloud-shell-create-storage.png "Azure Cloud Shell")
 
-   > **Note**: If creation fails, you may need to select **Advanced settings** and specify the subscription, region and resource group for the new storage account.
+   > **Note**: If creation fails, you may need to select **Advanced settings** and specify the subscription, region and resource group for the new storage account. This will create the new storage account for the shell.
```
Comment (kylebunting) [inline Hands-on lab/HOL step-by-step - App modernization.md:399]: This change is good and should be incorporated into the current lab guide.

File: Hands-on lab/HOL step-by-step - App modernization.md [hunk 2]
```diff
@@ -396,7 +396,9 @@ In this task, you use the Azure Cloud shell to retrieve the IP address of the Sq
...
+   Alternatively, you can create the new storage account on your own. In that case please use 'locally redundant zone', because other options are disabled by the policy.
```
Comment (kylebunting) [inline Hands-on lab/HOL step-by-step - App modernization.md:400]: This change is good and should be incorporated into the current lab guide.

File: Hands-on lab/HOL step-by-step - App modernization.md [hunk 3]
```diff
@@ -414,7 +416,10 @@ In this task, you use the Azure Cloud shell to retrieve the IP address of the Sq
    az vm list-ip-addresses -g $resourceGroup -n SqlServer2008 --output table
    ```
 
-   > **Note**: If you have multiple Azure subscriptions, and the account you are using for this hands-on lab is not your default account, you may need to run `az account list --output table` at the Azure Cloud Shell prompt to output a list of your subscriptions, then copy the Subscription Id of the account you are using for this lab, and then run `az account set --subscription <your-subscription-id>` to set the appropriate account for the Azure CLI commands.
+ > **Note**: The name of your VM with the SQL server is different if you work in the provisioned lab environment. Use ```-n Sql2008-*****```
```
Comment (kylebunting) [inline Hands-on lab/HOL step-by-step - App modernization.md:419]: This change is specific to using a pre-provisioned environment and should not be included in the pull request.

File: Hands-on lab/HOL step-by-step - App modernization.md [hunk 4]
```diff
@@ -414,7 +416,10 @@ In this task, you use the Azure Cloud shell to retrieve the IP address of the Sq
...
+ The right name can be obtained from the resource list in the Azure portal.
```
Comment (kylebunting) [inline Hands-on lab/HOL step-by-step - App modernization.md:420]: This change is specific to using a pre-provisioned environment and should not be included in the pull request.

File: Hands-on lab/HOL step-by-step - App modernization.md [hunk 5]
```diff
@@ -414,7 +416,10 @@ In this task, you use the Azure Cloud shell to retrieve the IP address of the Sq
...
+
+> **Note**: If you have multiple Azure subscriptions, and the account you are using for this hands-on lab is not your default account, you may need to run `az account list --output table` at the Azure Cloud Shell prompt to output a list of your subscriptions, then copy the Subscription Id of the account you are using for this lab, and then run `az account set --subscription <your-subscription-id>` to set the appropriate account for the Azure CLI commands.
```
Comment (kylebunting) [inline Hands-on lab/HOL step-by-step - App modernization.md:422]: This text is from the current workshop and does not appear to have been changed.

File: Hands-on lab/HOL step-by-step - App modernization.md [hunk 6]
```diff
@@ -812,6 +817,10 @@ In this task, you use the Azure Cloud Shell and Azure Command Line Interface (CL
 5. In the output table, locate the subscription you are using for this hands-on lab, and copy the SubscriptionId value into a text editor, such as Notepad, for use below.
 
 6. Next, enter the following `az ad sp create-for-rbac` command at the Cloud Shell prompt, replacing `<your-subscription-id>` with the value you copied above and `<your-resource-group-name>` with the name of your **hands-on-lab-SUFFIX** resource group, and then press `Enter` to run the command.
+This command sets the read permission on the resource group.
+>NOTE: If you are working in the lab environment, you might get following error:
```
Comment (kylebunting) [inline Hands-on lab/HOL step-by-step - App modernization.md:821]: This change is specific to using a pre-provisioned environment and should not be included in the pull request.

File: Hands-on lab/HOL step-by-step - App modernization.md [hunk 7]
```diff
@@ -812,6 +817,10 @@ In this task, you use the Azure Cloud Shell and Azure Command Line Interface (CL
 5. In the output table, locate the subscription you are using for this hands-on lab, and copy the SubscriptionId value into a text editor, such as Notepad, for use below.
 
 6. Next, enter the following `az ad sp create-for-rbac` command at the Cloud Shell prompt, replacing `<your-subscription-id>` with the value you copied above and `<your-resource-group-name>` with the name of your **hands-on-lab-SUFFIX** resource group, and then press `Enter` to run the command.
+This command sets the read permission on the resource group.
```
Comment (kylebunting) [inline Hands-on lab/HOL step-by-step - App modernization.md:820]: This change is good and should be incorporated into the current lab guide.

**Review by kylebunting — CHANGES_REQUESTED** [review]
@ddobric, please review my comments below, remove the few lines specific to using a pre-provisioned environment, and then I think we can pull in the other changes.

Thanks!

**Comment by timahenning** [comment]
Hi @ddobric,

Some of your changes in the submitted PR for the MS master update are great catches and can be accepted.  Kyle has made comments about some of the changes that need to be removed.  They cannot be accepted because they pertain to a pre-provisioned environment.  If you can make those recommended changes, we can get your updates merged into the master branch.  The other option is keep your changes in your local branch.

Let us know soon. Thanks.

Tim

**Comment by DawnmarieDesJardins** [comment]
@ddobric 
Hi Damir.  Thanks for looking at this.  Due to current issues, we are starting a refresh of this workshop that will incorporate your requests.  Please look for that to be complete mid-February.  For now, closing your PR to move forward with the workshop refresh.

**Pull request closed without merging**